### DISCUSSION TITLE
Wonder what's the IP of WebRTC config if frigate runs inside Docker container

### DISCUSSION THREAD

**Discussion by t0ny-peng** [blakeblackshear/frigate#6099] [Ask A Question] [answered]
Here's my setup. Host IP is `10.0.0.233`, and Frigate runs as a docker container with `network: bridge` and IP being 172.15.x.x, subject to change everytime it starts. [This section of the doc](https://docs.frigate.video/configuration/live#webrtc-extra-configuration) mentions that to get WebRTC working we need to set the IP. In this case, which IP should I use then?
Thx!

**Comment by t0ny-peng** [comment]
Btw, is this section correct? I thought `bridge` means a virtual linux bridge, which has its NAT on the host. What it describes seems to be macvlan, not bridge mode. 

> network: bridge creates a virtual network interface for the container, and the container will have full access to it. You also don't need to forward any ports, however, the IP for accessing Frigate locally will differ from the IP of the host machine. Your router will see Frigate as if it was a new device connected in the network.

**Reply by NickM-27** [reply]
You are right, I don't think that's correct.

**Reply by NickM-27** [reply]
I have made a PR to correct the docs

**Comment by CodingSquirrel** [comment, marked as answer]
It should be your host IP that you put there. That's what I have it set to and WebRTC is working for me.
